### PR TITLE
Debug; Closes #40;

### LINKED ISSUE #40: Provide a way to see all the API calls going over the wire
Feature request: As a CLI user I need to know the underlying API calls so that I can learn the reference information or debug why the command isn't working as expected.

### PR DESCRIPTION
**Pull request by jrperritt**

### CODE REVIEW

**Comment by kenperkins** [comment]
At first glance `--log info` and `--log debug` seem to do the same thing. Is that by design?

**Comment by jrperritt** [comment]
`info` logs a subset of `debug`. Try a command that has a request body like `rack networks network create --name testNet`.

File: internal/vendor.json [hunk 1]
```diff
@@ -1,269 +1,320 @@
...
+			"local": "github.com/jrperritt/rack/internal/github.com/rackspace/gophercloud/testhelper/client",
+			"revision": "f956c6c6c0c55844eff4b153b5071ef6e3ab4ab4",
+			"revisionTime": "2015-06-30T10:09:15-06:00"
+		},
+		{
+			"canonical": "gopkg.in/ini.v1",
+			"comment": "",
+			"local": "github.com/jrperritt/rack/internal/gopkg.in/ini.v1",
+			"revision": "177219109c97e7920c933e21c9b25f874357b237",
+			"revisionTime": "2015-03-19T00:52:36-04:00"
```
Comment (rgbkrk) [inline internal/vendor.json]: I'm assuming you have a different vendor version which triggered a migration of fields here?

**Comment by rgbkrk** [comment]
Looks good to me.

Does logrus always report to stdout? If we log with warnings or errors, does that go to stderr?

File: internal/vendor.json (hunk 1 above)
Comment (jrperritt) [inline internal/vendor.json]: Hmm, I hadn't noticed that. It looks like this new `vendor.json` has the correct format though (according to https://github.com/kardianos/vendor-spec#example)

**Comment by jrperritt** [comment]
Cool.

No, we can specify where the output goes. Currently I'm printing out to the writer from the CLI library, which we could eventually offer as a flag: https://github.com/jrperritt/rack/pull/171/files#diff-307e497b2df5889f375eae1979b2b8d9R225

**Comment by rgbkrk** [comment]
Ah, ok. No use in adding too many configurables, I wouldn't worry about a flag there just yet.

**Comment by jrperritt** [comment]
And I haven't   ;)

**Pull request merged**